README.md

```diff
@@ -0,0 +1,46 @@
+# Install OMERO on CentOS 7 or Ubuntu 18.04 using Ansible
+
+1. Install Ansible.
+   For instance, you can use a Python virtualenv::
+    ```
+    python3 -mvenv ansible-venv
+    . ansible-venv/bin/activate
+    python3 -mpip install ansible
+    ```
+   A conda environment
+    ```
+    conda create -n ansible ansible
+    conda activate ansible
+    ```
+   or some other package manager.
+2. Edit `ansible_host` in [`inventory.yml`](inventory.yml) to point to the server you wish to install OMERO on.
+3. Check you can SSH into the server, and that you can run `sudo`.
+4. Install the required Ansible roles:
+```
+ansible-galaxy install -p roles -r requirements.yml
+```
+5. Run `ansible-playbook`:
+```
+ansible-playbook -i inventory.yml playbook.yml
+```
+Use `playbook.yml` for CentOS 7 and `playbook-ubuntu.yml` for Ubuntu 18.04.
```

Comment:
how do you forsee the naming being when we add support for C8 or 20.04?

Reply:
Not completely sure yet. `playbook-ubuntu.yml` may also work for both Ubuntu 20.04 and also CentOS 8.